Mcu/g431/Src/phaseouts.c

```diff
@@ -61,7 +60,7 @@ void proportionalBrake()
 
 void phaseBPWM()
 {
-    if (!comp_pwm) { // for future
+    if (!eepromBuffer.comp_pwm) { // for future
```

Comment:
need eeprom.h include for phaseouts.c here

Reply:
added